md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -36,9 +36,9 @@ You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB
  * `HTTP`:
    The `HTTP` connection mode will connect the client to the engine using the http protocol. This is the most common way to connect to engine remotly.  
    When using `HTTP` mode, you must set the following system properties
-   * `org.bonitasoft.engine.api-type.server.url`
+   * `server.url`
```

Comment:
When using System properties, you MUST leave the prefix `org.bonitasoft.engine.api-type.`
Please fix this.